3684/0/triangle.rb

```diff
@@ -0,0 +1,39 @@
+# Triangle Project Code.
+
+# rubocop:disable Metrics/AbcSize
+# rubocop:disable Metrics/PerceivedComplexity
+# rubocop:disable Metrics/CyclomaticComplexity
+# Triangle analyzes the lengths of the sides of a triangle
+# (represented by a, b and c) and returns the type of triangle.
+#
+# It returns:
+#   :equilateral  if all sides are equal
+#   :isosceles    if exactly 2 sides are equal
+#   :scalene      if no sides are equal
+#
+# The tests for this method can be found in
+#   about_triangle_project.rb
+# and
+#   about_triangle_project_2.rb
+#
+
+# :reek:FeatureEnvy
+def triangle(aside, bside, cside)
+  array = [aside, bside, cside].sort
+  raise TriangleError if aside <= 0 || bside <= 0 || cside <= 0 || (array[0] + array[1] <= array[2])
```

Comment:
just 1 micro hint. rename `array` to `triangle_sides`. and user `triangle_sides.first, .second, .third`

how do you think, would it look better?

Reply:
thats true, its more communicative name in this case